packages/suite/src/views/wallet/anonymize/components/CoinjoinDefaultStrategy.tsx

```diff
@@ -29,36 +31,25 @@ const Text = styled(P)`
     margin: 20px 0;
 `;
 
-// Parameters sent to TrezorConnect.authorizeCoinjoin method
-// TODO: replace with dynamic values
-export const COINJOIN_STRATEGIES = {
-    recommended: {
-        maxRounds: 10,
-        targetAnonymity: 80,
-        maxFeePerKvbyte: 129000,
-        maxCoordinatorFeeRate: 0.003 * 10 ** 10, // 0.003 from coordinator
-    },
-    fast: {
-        maxRounds: 3,
-        targetAnonymity: 40,
-        maxFeePerKvbyte: 129000,
-        maxCoordinatorFeeRate: 0.003 * 10 ** 10, // 0.003 from coordinator
-    },
-};
+const kvB2vB = (v: number) => (v ? Math.round(v / 1000) : 1);
```

Comment:
magic function? 🪄  

At least a comment or a better name would be nice

Reply:
Moved and comment added in 93673b2e59367ed13545dc2e5117e9c1433d505d